src/test/java/picard/fingerprint/CrosscheckFingerprintsTest.java

```diff
@@ -178,10 +180,41 @@ public Object[][] bamAndCramFilesRGs() {
         };
     }
 
+    private Object[][] haplotypeMaps(){
+        return new Object[][]{{HAPLOTYPE_MAP}, {HAPLOTYPE_MAP_SHORT}};
+    }
+    @DataProvider(name = "bamFilesRGs")
```

Comment:
I think this has gotten a little twisted around.  You're testing the both full and short dictionary here, but only the full dictionary crams, and then you also have `testCrossCheckRGs_with_short_dictionary`, where you basically do the same thing, expect hardcoded instead of via the data provider.  I think that's also where the test failure is coming from, `bamFilesRGs` now includes the haplotypeMaps, but `testCrossCheckRGs_with_short_dictionary` doesn't include the haplotypeMaps as a parameter input.

Reply:
😊  huh. I saw other errors before. yes, something got strange.